saplings/circuits/src/components/circuitDetails/CircuitDetails.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../Content.scss';
+import '../MainHeader.scss';
+import './CircuitDetails.scss';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faQuestionCircle,
+  faArrowLeft
+} from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+import { useParams, Link } from 'react-router-dom';
+import React from 'react';
+
+import { Circuit } from '../../data/circuits';
+import { useCircuitState } from '../../state/circuits';
+import { getNodeRegistry } from '../../api/splinter';
+
+const CircuitDetails = () => {
+  const { circuitId } = useParams();
+  const [circuit] = useCircuitState(circuitId);
+
+  if (!circuit) {
+    return <div />;
+  }
+  return (
+    <div>
+      <div className="main-header">
+        <div className="back-button">
+          <Link to="/circuits">
+            <FontAwesomeIcon icon={faArrowLeft} />
+            &nbsp;Circuits
+          </Link>
+        </div>
+        <div className="circuits-title">
+          <h4>
+            Circuit
```

Comment:
There should be a space between the word circuit and the circuit ID. Right now they display as a single word:
<img width="531" alt="Screen Shot 2020-06-18 at 7 44 59 AM" src="https://user-images.githubusercontent.com/14094978/85021578-a5ba7780-b137-11ea-83c8-f5d7d9674a9c.png">

Reply:
Fixed